src/Common/src/Interop/Windows/kernel32/Interop.SetThreadErrorMode.Uap.cs

```diff
@@ -8,10 +8,11 @@ internal partial class Interop
 {
     internal partial class Kernel32
     {
-        internal static uint SetErrorMode(uint uMode)
+        internal static bool SetThreadErrorMode(uint dwNewMode, out uint lpOldMode)
         {
             // Prompting behavior no longer occurs in all platforms supported
-            return 0;
+            lpOldMode = 0;
```

Comment:
Fyi: This may not, in fact, be true. According to initial discussions with FS devs this behavior still exists. I'm trying to find a setup with a physical floppy (not usb) to test this.